website/blog/2023-11-15-run-webassembly-wasm-workloads-windows-and-macos.md

```diff
@@ -0,0 +1,339 @@
+info: default host triple is aarch64-unknown-linux-gnu
+info: syncing channel updates for 'stable-aarch64-unknown-linux-gnu'
+info: latest update on 2023-10-05, rust version 1.73.0 (cc66ad468 2023-10-03)
+info: downloading component 'cargo'
+info: downloading component 'clippy'
+info: downloading component 'rust-docs'
+info: downloading component 'rust-std'
+info: downloading component 'rustc'
+info: downloading component 'rustfmt'
+info: installing component 'cargo'
+info: installing component 'clippy'
+info: installing component 'rust-docs'
+info: installing component 'rust-std'
+info: installing component 'rustc'
+info: installing component 'rustfmt'
+info: default toolchain set to 'stable-aarch64-unknown-linux-gnu'
+
+  stable-aarch64-unknown-linux-gnu installed - rustc 1.73.0 (cc66ad468 2023-10-03)
+
+
+Rust is installed now. Great!
+
+To get started you may need to restart your current shell.
+This would reload your PATH environment variable to include
+Cargo's bin directory ($HOME/.cargo/bin).
+
+To configure your current shell, run:
+source "$HOME/.cargo/env"
+info: downloading component 'rust-std' for 'wasm32-wasi'
+info: installing component 'rust-std' for 'wasm32-wasi'
+--> c93a3433d432
+[1/2] STEP 3/6: COPY Cargo.toml /app/
+--> cf4488993835
+[1/2] STEP 4/6: COPY src /app/src
+--> 531b9389857c
+[1/2] STEP 5/6: WORKDIR /app
+--> 23379392f585
+[1/2] STEP 6/6: RUN source "$HOME/.cargo/env" && cd /app && cargo build --target wasm32-wasi --release
+   Compiling rust-hello-world v0.1.0 (/app)
+    Finished release [optimized] target(s) in 0.15s
+--> e3582e06f45b
+[2/2] STEP 1/3: FROM scratch
+[2/2] STEP 2/3: ENTRYPOINT [ "/rust-hello-world.wasm" ]
+--> 069b1742d906
+[2/2] STEP 3/3: COPY --from=builder /app/target/wasm32-wasi/release/rust-hello.wasm /rust-hello-world.wasm
+[2/2] COMMIT rust-hello-world-wasm
+--> e0948298c0be
+Successfully tagged localhost/rust-hello-world-wasm:latest
+e0948298c0be20e11da5d92646a2d6453f05e66671f72f0f792c1e1ff8de75ba
+```
+
+This is a multi-stage build but at the end we only have a small image containing the wasm binary.
+
+Launch it quickly using
+
+```shell-session
+$ podman run rust-hello-world-wasm
+```
+
+and we'll see the expected output
+
+```
+WARNING: image platform (wasi/wasm/v8) does not match the expected platform (linux/arm64)
+
+!... Hello Podman wasm World ...!
+
+         .--"--.
+       / -     - \
+      / (O)   (O) \
+   ~~~| -=(,Y,)=- |
+    .---. /`  \   |~~
+ ~/  o  o \~~~~.----. ~~
+  | =(X)= |~  / (O (O) \
+   ~~~~~~~  ~| =(Y_)=-  |
+  ~~~~    ~~~|   U      |~~
+
+Project:   https://github.com/containers/podman
+Website:   https://podman.io
+Documents: https://docs.podman.io
+Twitter:   @Podman_io
+
+```
+
+​
+
+## Conclusion
+
+Now, you've seen that running or building wasm workloads is possible on Windows and macOS when using podman.
+So it's up to you to try to run and build new examples and report problems to the podman community if you are facing issues.
```

Comment:
Could use a stronger and more positive call to action, e.g. something like 'WASM is growing in popularity and more applications are being packaged as WASM every day - give it a try yourself and feel free to reach out to the Podman community if you have issues!'